exo-go/src/docker_setup/index_test.go

```diff
@@ -0,0 +1,74 @@
+package dockerSetup_test
+
+import (
+	"path"
+	"regexp"
+
+	"github.com/Originate/exosphere/exo-go/src/app_config_helpers"
+	"github.com/Originate/exosphere/exo-go/src/docker_setup"
+	"github.com/Originate/exosphere/exo-go/src/logger"
+	"github.com/Originate/exosphere/exo-go/src/service_config_helpers"
+	"github.com/Originate/exosphere/exo-go/src/types"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("GetServiceDockerConfigs", func() {
+
+	var setup *dockerSetup.DockerSetup
+	var appDir string
+
+	var _ = BeforeSuite(func() {
+		appDir = path.Join("..", "..", "..", "exosphere-shared", "example-apps", "external-dependency")
+		appConfig, err := appConfigHelpers.GetAppConfig(appDir)
+		Expect(err).NotTo(HaveOccurred())
+		serviceConfigs, err := serviceConfigHelpers.GetServiceConfigs(appDir, appConfig)
+		Expect(err).NotTo(HaveOccurred())
+		serviceData := serviceConfigHelpers.GetServiceData(appConfig.Services)
+		serviceName := "mongo"
+		setup = dockerSetup.NewDockerSetup(appConfig, serviceConfigs[serviceName], serviceData[serviceName], serviceName, logger.NewLogger([]string{}, []string{}), appDir)
```

Comment:
Please create a new `dockerSetup` in a beforeEach so we have a new instance in each test. The rest should be fine to keep in `BeforeSuite `